lib/helpers.js

```diff
@@ -192,6 +192,8 @@ module.exports = function (APIWrapper) {
           array.push(arrayItem)
         }
         result[key] = array
+      } else if (typeof result[key] === 'boolean') {
+        // do nothing
```

Comment:
```
      } else if (typeof object[key] === 'boolean') {
        result[key] = object[key]

```

No?

Reply:
We could do that, yes. I'll fix it.

Reply:
I think if you do nothing ,you get `filter={"tmdbId":{}}`